src/aeso_parser.erl

```diff
@@ -596,8 +604,13 @@ expand_includes(AST, Included, Opts) ->
              || File <- lists:usort(auto_imports(AST)) ] ++ AST,
     expand_includes(AST1, Included, [], Opts).
 
-expand_includes([], _Included, Acc, _Opts) ->
-    {ok, lists:reverse(Acc)};
+expand_includes([], Included, Acc, Opts) ->
+    case lists:member(keep_included, Opts) of
```

Comment:
`Opts` have type `aeso_compiler:options()` and that does not include `keep_included`!?

Reply:
Good catch, will add